### ISSUE TITLE
Some documentation ?

### ISSUE THREAD

**Issue by dandumit** [triffid/LPC17xx-DFU-Bootloader#2]
Hello Triffid !

Thank you for your great work. I would like to know if there it's some documentation.
I saw trough code that bootloader can be loaded trough USB. Could you please provide some more details for a Windows user ?

Kind REgards,
Daniel

**Comment by pjar82** [comment]
Have u found if u can flash bootloader by usb yet

**Comment by triffid** [comment]
It's possible but I've never tried it.

I guess you'd have to change the linker script and compile dfu bootloader as an application, change the USER_FLASH_START and USER_FLASH_END to the original bootloader, then flash it via dfu, then flash the new proper bootloader via this franken-app-dfu

**Comment by fex42** [comment]
I agree to the point that a bit of documentation (a README.md would be good enough) would be good. Decribing what it is, how to build and install (even if windows is not important for me).

**Comment by triffid** [comment]
Added a README, feel free to send pulls if you'd like to improve/expand on it because I know it's very basic right now ;)
